web/libs/editor/src/stores/AppStore.js

```diff
@@ -561,6 +581,8 @@ export default types
         entity.sendUserGenerate();
       }
       handleSubmittingFlag(async () => {
+        await self.waitForDraftSubmission();
```

Comment:
Just curious, don't we need the same on updating, and accepting at least (there is also reject, but I do not think it does something with draft)

Reply:
the problem here is that submit doesn't know about draft because it's being saved in parallel and draft would not be associated with the annotation. all other actions with drafts will have associated annotation so can't be broken in this way.

Reply:
@hlomzik  Oh! Thanks. That helped a lot for understanding this and other issue.